content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,89 @@ description: Istio 1.5 release notes.
+- **Improved** fix readiness probe inconsistency #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings per host #18406.
+- **Fixed** an issue where pods crashing would trigger excessive configuration pushes #18574.
+- **Fixed** issues with applications that call themselves #19308.
+- **Added** detection of `iptables` failure when using Istio CNI #19534.
+- **Added** `consecutive_5xx` and `gateway_errors` as outlier detection options #19771.
+- **Improved** `EnvoyFilter` matching performance optimization #19786.
+- **Added** support for `HTTP_PROXY` protocol #19919.
+- **Improved** `iptables` setup to use `iptables-restore` by default #18847.
+- **Enabled** [automatic protocol detection](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection-experimental) by default #18666.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthentication` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** auto mutual TLS. This beta feature is enabled by default.
+- **Improved** node agent by merging with Pilot agent. This improves the security posture by removing the requirement of a pod security policy.
+- **Improved** Istio by including certificate provisioning functionality within Pilot.
+- **Improved** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters that [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
+- **Improved** Istio's Pilot Agent by provisioning key and certificates for Prometheus consumption.
+- **Graduated** SDS which provides identity provisioning for Istio.
+
+## Telemetry
+
+- **Added** TCP protocol support for v2 telemetry.
+- **Added** gRPC response status code support in metrics/logs.
+- **Added** support for Istio Canonical Service.
+- **Improved** stability of v2 telemetry pipeline.
+- **Added** alpha-level support for configurability in v2 telemetry.
+- **Added** support for populating AWS platform metadata in Envoy node metadata.
+- **Improved** Stackdriver adapter for Mixer to support configurable flush intervals for tracing data.
+- **Added** support for a headless collector service to the Jaeger addon.
+- **Fixed** `kubernetesenv` adapter to provide proper support for pods that contain a dot in their name.
+- **Improved** the Fluentd adapter for Mixer to provide millisecond-resolution in exported timestamps.
```

Comment:
is this relevant?  Mixer is not enabled by default.  We should definitely mention in release note that mixer-telemetry or policy is not enabled by default.

Reply:
We should mention blackholecluster is broken with telemetry - https://github.com/istio/istio/issues/21385

Reply:
It is relevant to the users that specifically added that feature. There is a small subset of users that will continue to use Mixer until the functionality is ported over. There are multiple changes here for Mixer-based functionality. We still support Mixer.

Having the release notes about the conversion to v2 telemetry not being 100% feature compatible with v1 seems more appropriate than saying "blackhole is broken", as blackhole telemetry still works in v1.  But I'm not sure having that in the change notes is the appropriate place.

Reply:
That is reasonable, I was trying to think how we can group the v2 telemtry and v1 telemetry without generating confusion to users.  Should we divide the bullets under telemetry for v2 and v1?  currently it is not clear unless it is specifically called out.